wolfssl/openssl/asn1.h

```diff
@@ -24,6 +24,9 @@
 #ifndef WOLFSSL_ASN1_H_
 #define WOLFSSL_ASN1_H_
 
+#ifdef WOLFSSL_WPAS
+#include <stddef.h>  /* for offsetof */
```

Comment:
offsetof is not portable for old embedded compilers.
Sorry, you'll have to find another way to do this.

Reply:
It is possible to define `offsetof` as `((size_t)((char *)&((st *)0)->m - (char *)0))` would this be fine? An alternative would be to declare a dummy static struct and calculate the offset based on that struct. For now I implemented the second method and it seems to work. Let me know what you think.

Reply:
When I looked around this code I can see that the offset is required.
So, defining offsetof when not available is fine.

Reply:
Use a dummy static struct (I presume you mean a static const global variable).
I have a vague memory of compilers not liking NULL.